.travis.yml

```diff
@@ -0,0 +1,85 @@
+language: php
+
+sudo: false
+
+addons:
+  postgresql: "9.5"
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.npm
+
+php:
+  - 7.1
+  - 7.2
+
+services:
+  - mysql
+
+env:
+  - DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE CI_PLUGIN=2
+  - DB=pgsql MOODLE_BRANCH=MOODLE_36_STABLE CI_PLUGIN=2
+  - DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE CI_PLUGIN=2
+  - DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE CI_PLUGIN=2
+  - DB=pgsql MOODLE_BRANCH=master CI_PLUGIN=2
+  - DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE CI_PLUGIN=2
+  - DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE CI_PLUGIN=2
+  - DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE CI_PLUGIN=2
+  - DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE CI_PLUGIN=2
+  - DB=mysqli MOODLE_BRANCH=master CI_PLUGIN=2
+
+matrix:
+  exclude:
+    - php: 7.1
+      env: DB=mysqli MOODLE_BRANCH=master CI_PLUGIN=2
+    - php: 7.1
+      env: DB=pgsql MOODLE_BRANCH=master CI_PLUGIN=2
+  include:
+    - php: 7.3
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE CI_PLUGIN=2
+    - php: 7.3
+      env: DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE CI_PLUGIN=2
+    - php: 7.3
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE CI_PLUGIN=2
+    - php: 7.3
+      env: DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE CI_PLUGIN=2
+    - php: 7.3
+      env: DB=mysqli MOODLE_BRANCH=master CI_PLUGIN=2
+    - php: 7.3
+      env: DB=pgsql MOODLE_BRANCH=master CI_PLUGIN=2
+
+before_install:
+  - phpenv config-rm xdebug.ini
+  - cd ../..
+  - echo $CI_PLUGIN
+  - git clone --mirror 'https://github.com/moodlehq/moodle-behat-extension.git' '/home/travis/.composer/cache/vcs/git-github.com-moodlehq-moodle-behat-extension.git/' || true
+  - if [ "$CI_PLUGIN" = 2 ]; then
```

Comment:
I think everything in the matrix above is CI_PLUGIN=2, so we can probably just remove all this logic and only do the v2 testing

Reply:
Hi @brendanheywood 
This is removed in commit c9d1841e22374e2e6d31df555e05f79bc195e872